packages/overlays/src/OverlayController.js

```diff
@@ -7,20 +7,50 @@ import { containFocus } from './utils/contain-focus.js';
 /**
  * @typedef {import('../types/OverlayConfig').OverlayConfig} OverlayConfig
  * @typedef {import('../types/OverlayConfig').ViewportConfig} ViewportConfig
- * @typedef {import('popper.js').default} Popper
- * @typedef {import('popper.js').PopperOptions} PopperOptions
- * @typedef {{ default: Popper }} PopperModule
+ * @typedef {import('@popperjs/core/lib/popper').createPopper} Popper
+ * @typedef {import('@popperjs/core/lib/popper').Options} PopperOptions
+ * @typedef {import('@popperjs/core/lib/enums').Placement} Placement
+ * @typedef {{ createPopper: Popper }} PopperModule
  * @typedef {'setup'|'init'|'teardown'|'before-show'|'show'|'hide'|'add'|'remove'} OverlayPhase
  */
 
 /**
  * @returns {Promise<PopperModule>}
  */
 async function preloadPopper() {
-  // @ts-ignore
-  return /** @type {Promise<PopperModule>} */ (import('popper.js/dist/esm/popper.min.js'));
+  // @ts-ignore import complains about untyped module, but we typecast it ourselves
+  return /** @type {Promise<PopperModule>} */ (import('@popperjs/core/dist/esm/popper.js'));
 }
 
+// /**
+//  * @returns {Promise<PopperModule>}
+//  */
+// async function preloadPopper() {
```

Comment:
can we delete this comment block? 🤔

Reply:
@tlouisse  what was the purpose of this thing? we need it?

Reply:
No, I pushed a wip state. I tried to lazy load popper-lite including just the modifiers we need. However, the arrow modifier didn't seem to work properly with the code below.

So we can remove it